designer/client/src/components/graph/GraphPartialsInTS/applyCellChanges.ts

```diff
@@ -1,26 +1,33 @@
 /* eslint-disable i18next/no-literal-string */
-import { dia } from "jointjs";
+import type { Theme } from "@mui/material";
+import type { dia } from "jointjs";
 import { flatMap, groupBy, isEqual } from "lodash";
-import { ScenarioGraph, ProcessDefinitionData } from "../../../types";
+import { partition } from "lodash";
+
+import type { ScenarioGraph, ProcessDefinitionData } from "../../../types";
+import { StickyNoteType } from "../../../types/stickyNote";
 import { makeElement, makeLink } from "../EspNode";
+import type { ModelWithTool } from "../EspNode/stickyNoteElements";
+import { makeStickyNoteElement } from "../EspNode/stickyNoteElements";
 import NodeUtils from "../NodeUtils";
 import { isEdgeConnected } from "./EdgeUtils";
 import { updateChangedCells } from "./updateChangedCells";
-import { Theme } from "@mui/material";
-import { StickyNote } from "../../../common/StickyNote";
-import { makeStickyNoteElement, ModelWithTool } from "../EspNode/stickyNoteElements";
 
 export function applyCellChanges(
     paper: dia.Paper,
     scenarioGraph: ScenarioGraph,
-    stickyNotes: StickyNote[],
     processDefinitionData: ProcessDefinitionData,
     theme: Theme,
 ): void {
     const graph = paper.model;
 
-    const nodeElements = NodeUtils.nodesFromScenarioGraph(scenarioGraph).map(makeElement(processDefinitionData, theme));
-    const stickyNotesModelsWithTools: ModelWithTool[] = stickyNotes.map(makeStickyNoteElement(processDefinitionData, theme));
+    const [stickyNoteElements, scenarioNodeElements] = partition(
```

Comment:
wasn't this merged elsewhere only to be separated again here? what's the benefit?